aspnet/fundamentals/middleware.rst

```diff
@@ -10,12 +10,12 @@ In this article:
 	- `Built-in middleware`_
 	- `Writing middleware`_
 
-`Download sample from GitHub <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/middleware/sample>`_. 
+`Download sample from GitHub <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/middleware/sample>`_.
 
 What is middleware
 ------------------
 
-Middleware are components that are assembled into an application pipeline to handle requests and responses. Each component can choose whether to pass the request on to the next component in the pipeline, and can perform certain actions before and after the next component in the pipeline. Request delegates are used to build this request pipeline, which are then used to handle each incoming HTTP request to your application. 
+Middleware are components that are assembled into an application pipeline to handle requests and responses. Each component can choose whether to pass the request on to the next component in the pipeline, and can perform certain actions before and after the next component in the pipeline. Request delegates are used to build this request pipeline, which are then used to handle each incoming HTTP request to your application.
```

Comment:
Request delegates are used to build  the request pipeline. The request delegates handle each HTTP request to your app.
----- Commentary - no such thing as an outgoing HTTP request to your app -is there? That makes incoming redundant dead wood -- end of commentary..